src/main/java/org/wildfly/security/ssl/MechanismDatabase.properties

```diff
@@ -51,6 +51,10 @@ TLS_RSA_WITH_AES_128_CBC_SHA256         = AES128-SHA256,RSA,RSA,AES128,SHA256,TL
 TLS_RSA_WITH_AES_256_CBC_SHA256         = AES256-SHA256,RSA,RSA,AES256,SHA256,TLSv1.2,false,HIGH,true,256,256,00,3D
 TLS_RSA_WITH_AES_128_GCM_SHA256         = AES128-GCM-SHA256,RSA,RSA,AES128GCM,AEAD,TLSv1.2,false,HIGH,true,128,128,00,9C
 TLS_RSA_WITH_AES_256_GCM_SHA384         = AES256-GCM-SHA384,RSA,RSA,AES256GCM,AEAD,TLSv1.2,false,HIGH,true,256,256,00,9D
+TLS_RSA_WITH_AES_128_CCM                = AES128-CCM,RSA,RSA,AES128CCM,AEAD,TLSv1.2,false,HIGH,false,128,128,C0,9C
```

Comment:
Minor nit - the 4 newly added encryption names should also be added to the "Valid enc names" list at the top of this file. Any other new values that have been added for the other parameters should also be added at the top of this file.

Reply:
Added AES128CCM, AES128CCM8, AES256CCM, AES256CCM8 to valid enc names and EDH to valid kex names.